api/apps/geoprocessing/src/modules/scenario-planning-units-features-aggregate/scenario-planning-units-features-aggregate-processor.ts

```diff
@@ -24,8 +27,8 @@ species AS (
 UPDATE scenarios_pu_data
 SET feature_list = sub.feature_list
 FROM (SELECT pu.scenario_id,
-             pu.id                                        AS scenario_pu_id,
-             ARRAY_AGG(DISTINCT species.feature_id::text) AS feature_list
+             pu.id AS scenario_pu_id,
+             ARRAY_AGG(DISTINCT concat_ws(':',species.feature_id::text,pu.amount)) AS feature_list
```

Comment:
Are we sure we want to let go of the "plain" feature_list (without amounts)?

Thinking about use cases such as showing "how many features are present, irrespective of their amount, in any given PU", I'd lean on keeping that.

For the specific use case of counts of features per PU in practice that could be even compressed to a plain count, not even to an array of the UUIDs of the features present in the PU, and this could be easily done at query time, or through a generated column that holds the count of items in `feature_list`.

tl;dr should we `insert` the data above into a new column such as `feature_and_amount_list` (this being an array of `[uuid, float]` pairs, as a new `CREATE TYPE feature_amounts AS (feature_id UUID, amount FLOAT)`), and then keep storing the plain list of feature uuids into the existing `feature_list` column, and then if we want/need, add a new column such as `feature_count int generated always as (cardinality(feature_list)) stored`?